lib/pengine/pe_output.c

```diff
@@ -3203,6 +3252,22 @@ ticket_xml(pcmk__output_t *out, va_list args) {
         free(buf);
     }
 
+    if (details) {
```

Comment:
xml can always add details (unless we need it not to somewhere for backward compatibility)

Reply:
Fixed.  The only catch here was making sure we don't also output a `granted=` attribute, since that duplicates the status attribute.  While I don't think that causes any problems, nothing was outputting it previously.